protocol/rep/rep.go

```diff
@@ -227,9 +225,8 @@ func (r *rep) RecvHook(m *mangos.Message) bool {
 	}
 	r.sock.SetSendError(nil)
 	r.backtraceL.Lock()
-	r.backtrace = append(r.backtracebuf[0:0], m.Header...)
+	r.backtrace = append(r.backtrace[0:0], m.Header...)
 	r.backtraceL.Unlock()
-	m.Header = nil
```

Comment:
Yes, and if we do this, we need to make a change to REQ/REP in nanomsg too.  This would be a wire protocol change, so we have to change both places.  I need to have folks look at this as well.